### DISCUSSION TITLE
Distro version

### DISCUSSION THREAD

**Discussion by happycodinguk** [alfiosalanitri/wireguard-client-connection-notification#3] [General]
Hello.
Am liking the look of this script.
Which distro is it tested on as I can't currently get much out of it.

Cheers!

**Comment by alfiosalanitri** [comment]
Hi, thanks.
I have tested it only with Raspbian 10 (buster).
But I think that it works with all Debian based distro.

**Reply by happycodinguk** [reply]
Hello again! 

Useful piece of script there. Love it.
I reinstalled wireguard via PIVPN (not the manual install I first had) and all was well.

Thanks for your time on creating this.
Appreciated!
